src/LaraguardServiceProvider.php

```diff
@@ -45,6 +44,14 @@ public function boot(Repository $config, Router $router, Dispatcher $dispatcher)
             $this->publishes([
                 __DIR__ . '/../resources/views' => resource_path('views/vendor/laraguard'),
             ], 'views');
+
+            if (! class_exists('CreateTwoFactorAuthenticationsTable')) {
```

Comment:
I assume that once the migration is published, it will create the  `CreateTwoFactorAuthenticationsTable`  class, and the migration publishing won't work for a second time. Is that correct?

Reply:
Yes, once the migration is published, then the `CreateTwoFactorAuthenticationsTable` will exist preventing it from publishing the migration again.